### PR TITLE
fix: ipc send messages

### PR DESCRIPTION
**Pull request by vashian**

### CODE REVIEW

File: src/commands/deploy.ts [hunk 1]
```diff
@@ -419,18 +449,23 @@ To file a ticket, please head to: https://console.liara.ir/tickets`)
...
               }
+              ipcSendLog({log: 'Release failed.', state: 'build', status: 'error'})
               return reject(new Error('Release failed.'))
             }
 
             if (release.state === 'DEPLOYING' && !isDeploying) {
               isDeploying = true
+              ipcSendLog({log: 'Image pushed.', state: 'build', status: 'pending'})
               this.spinner.succeed('Image pushed.')
+              ipcSendLog({log: 'Creating a new release...', state: 'build', status: 'pending'})
```
Comment (mhemrg) [inline src/commands/deploy.ts:463]: I guess we need to remove this line.

File: src/commands/deploy.ts [hunk 2]
```diff
@@ -620,6 +668,8 @@ To file a ticket, please head to: https://console.liara.ir/tickets`)
       const packageJSON = fs.readJSONSync(path.join(projectPath, 'package.json'))
 
       if (!packageJSON.scripts || !packageJSON.scripts.start) {
+        ipcSendLog({log:`A NodeJS app must be runnable with 'npm start'.
+        You must add a 'start' command to your package.json scripts.`, state: 'prebuild', status: 'error'})
```
Comment (mhemrg) [inline src/commands/deploy.ts]:
```suggestion
        You must add a 'start' command to your package.json scripts.`, state: 'pre-build', status: 'error'})
```

File: src/commands/deploy.ts (hunk 1 above)
Comment (vashian) [reply src/commands/deploy.ts:463]: Literally, I don't know why  
we are supposed to show users' new releases, aren't we??

Comment (mhemrg) [reply src/commands/deploy.ts:463]: We're printing `Creating a new release...` log in 2 different lines. You can `ctrl + f` it.

Comment (vashian) [reply src/commands/deploy.ts]: you were right... they were in differentness stage

**Comment by mhemrg** [comment]
The first beta version of this PR has been published:

```
npm i @liara/cli@2.21.3-beta.0
```

**Pull request closed without merging**